app/policies/comment_policy.rb

```diff
@@ -0,0 +1,10 @@
+class CommentPolicy < ApplicationPolicy
+  attr_reader :user, :comment
+
+  def destroy?
+    user.present?
+  end
+  def create?
+    user.present?
+  end
```

Comment:
As Thierry mentioned, would be good practice to move the check on who can take action on comments from Controller to here in Policy. Would want a policy on who can update comments as well, not just create and destroy